app.js

```diff
@@ -470,6 +470,11 @@ app.post(
   storyController.commentOnCommentSubmit
 );
 
+app.post(
```

Comment:
Why the use of post instead of put here? I interpret editing as an updating operation.